src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-and-dt.mdx

```diff
@@ -25,10 +23,7 @@ Infinite Tracing is not yet available for mobile monitoring.
 
 For mobile agents that support this feature, it’s enabled by default. 
 
-If you prefer to turn off distributed tracing, see the following:
-
-    * [Android feature flag](/docs/mobile-monitoring/new-relic-mobile-android/api-guides/android-agent-configuration-feature-flags/#dt)
-    * [iOS feature flag](/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags/#dt)
+If you prefer to turn off distributed tracing, see the following: [iOS feature flag](/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags/#dt)
```

Comment:
I'd go with: "If you prefer to turn off distributed tracing, see [iOS feature flag](/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags/#dt)"

Reply:
@urbiz-nr, good suggestion!